data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+		"topic": "BGSS_OUT_OF_TOWN_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Out_of_Town_1" } 
+	},
+	{ "text": "Tell me about how you survived (OutTown2).", 
+		"topic": "BGSS_OUT_OF_TOWN_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Out_of_Town_2" } 
+	},
+	{ "text": "Tell me about how you survived (LostPart1).", 
+		"topic": "BGSS_LOST_PARTNER_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Lost_Partner_1" } 
+	},
+	{ "text": "Tell me about how you survived (LostPart2).", 
+		"topic": "BGSS_LOST_PARTNER_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Lost_Partner_2" } 
+	}
+
+
+
+
+
+
+
+
+     ]
+},
+{
+    "id": "BGSS_CONFUSED_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "I don't even fucking know anymore. I have no fucking idea what is going on. I'm just doing what I can to stay alive. World ended and I bungled along not dying, until I met you.",
+    "responses": [ { "text": "Huh.", "topic": "TALK_FRIEND" } ]
+},
+{
+    "id": "BGSS_NO_PAST_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "Like, what? Before the apocalypse? Who cares about that? This is a new world, and yeah, it's pretty shitty. It's the one we've got though, so let's not dwell in the past when we should be making the best of what little we have left.",
+    "responses": [
+      { "text": "I can respect that.", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_NO_PAST_2_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "To be honest... I don't really remember. I remember vague details of my life before the world was like this, but the actual end of the world? It's all a blur. I don't know how I got where I am now, or how any of this happened. I think something pretty bad must have happened to me. Or maybe I was just hit in the head really hard. Or both. Both seems likely.",
+    "responses": [
+      { "text": "Huh.", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_RELIGIOUS_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "My story. Huh. It's nothing special. I had people, but they've risen to be with the Lord. I don't understand why He didn't take me too, but I suppose it'll all be clear in time.",
+    "responses": [
+      { "text": "Do you mean in a religious sense, or...?", "topic": "BGSS_RELIGIOUS_1_STORY2" },   
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_RELIGIOUS_1_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "Of course. It's clear enough, isn't it? That... that end, was the Rapture. I'm still here, and I still don't understand why, but I will keep Jesus in my heart through the Tribulations to come. When they're past, I'm sure He will welcome me into the Kingdom of Heaven. Or... or something along those lines. It's not going exactly like I thought it would, but that's prophecy for you.",
+    "responses": [
+      { "text": "What if you're wrong?", "topic": "BGSS_RELIGIOUS_1_FAITH1" },
```

Comment:
Consider reducing their opinion of the player, something like:
```
{ "text": "What if you're wrong?", "topic": "BGSS_RELIGIOUS_1_FAITH1", "condition": { "not": { "npc_has_effect": "player_rude_doubts" } }, "effect": { "npc_add_effect": "player_rude_doubts", "duration": "PERMANENT", "opinion": { "value": -1 } },
{ "text": "What if you're wrong?", "topic": "BGSS_RELIGIOUS_1_FAITH1", "condition": { "npc_has_effect": "player_rude_doubts" } },
```
so the player can't repeat the story and tank the opinion, but the NPC is still mildly insulted.

Reply:
Agreed. In particular that's upcoming if players try to persuade someone to tell them a tough story and fail. I just haven't added it yet